site/en/guides/border-router/_toc.yaml

```diff
@@ -1,12 +1,14 @@
 toc:
 - title: Overview
   path: /guides/border-router/
-- heading: Platforms
+- heading: Platforms - Linux
 - title: BeagleBone Black
   path: /guides/border-router/beaglebone-black
 - title: Raspberry Pi
   path: /guides/border-router/raspberry-pi
-- heading: Setup
+- heading: Platforms - FreeRTOS
+- title: Espressif ESP32 Series
+  path: /guides/border-router/espressif-esp32
 - title: Build and Configuration
```

Comment:
Oh, add back the Setup heading please

Before the "Build and Configuration" line should be

```
- heading: Setup
```